modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/content_editor/components/AIAssistentChat/AIAssistentChat.tsx

```diff
@@ -0,0 +1,249 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import React, {useRef, useState} from 'react';
+
+import AssistentMessageBalloon from './components/AssistentMessageBalloon';
+import UserMessageBalloon from './components/UserMessageBalloon';
+
+import './chat.scss';
+
+interface message {
+	sender: string;
+	text: string;
+}
+
+const AIAssistentChat: React.FC = () => {
```

Comment:
@DavysonMelo rename to `AIAssistantChat`, for each place that has `assistent` rename to `assistant`.

Thanks!